packages/bson-transpilers/test/yaml/partial.yaml

```diff
@@ -175,6 +188,7 @@ tests:
           csharp: DateTime
           shell: Date
           ruby: Time
+          rust: Utc
```

Comment:
ditto using `bson::DateTime` instead of a chrono type

Reply:
Done.